### ISSUE TITLE
Quickstart page is now missing core docs eg "Restrictions" and "Known Issues"

### ISSUE THREAD

**Issue by adamgit** [Unity-Technologies/PostProcessing#269]
##### What happened?

v1 core docs which were embedded in the Quickstart (shouldnt be there IMHO, should be separate pages on wiki - then there would be no problem!) were deleted. Finding restrictions - e.g. that TAA won't work with VR - requiers you to ignore the v2 docs and instead go read the v1 docs, "guessing" that the QuickStart guide will (rather surprisingly) contain core items like "X doesn't work with VR".

The links in the forum, even for v2, just go to the front page of the wiki, giving no indication that you need to open the v1 (not v2!) quickstart to find what you're looking for.

##### Post-processing stack version (v1, v2)?

v2

##### Unity version, operating system, target platform (standalone windows, mac, iOS, PS4...)?

n/a

**Comment by Chman** [comment]
The documentation on Github isn't official. It is, like the title say, just a quick start guide and a few random pages about some specific features.

> were deleted

Nothing has been deleted. Initially it's always been a single quickstart page for v1 with everything on it (restrictions & known issues included). It's still the case. The only thing that changed is the addition of v2 docs and because of that an index page. I'm not sure what's not clear about [this page](https://github.com/Unity-Technologies/PostProcessing/wiki), you can clearly see a separation between v1 and v2 docs. Then again, these are still quick extra docs until the Unity manual gets updated (which already has proper documentation for v1).

**Comment by adamgit** [comment]
1. All Unity's links for the tech point to the wiki. It doesn't really matter whether you call it "official" or not - it's what Unity Tech is linking to, so for now ... it's what people are using.
2. The key info is missing from the v2 quickstart. The headings are missing. You can say it's "not deleted" if you like, but the info is required, was present in v1, and is missing from v2.

Call it what you like - I don't mind :) - but closing this is bizarre. The info is needed, and is missing. Fix it however you like (if I had access, personally I'd refactor the docs/release notes to make it easier to maintain - split the restrictions and known issues into a different wiki page so that you don't have to do any further work on it!).

But don't just deny the problem that's right in front of us :).

**Comment by adamgit** [comment]
To be clear: Unity's official pages on VR link directly to this wiki.

So ... maybe you told Unity staff it's not official, but they've misunderstood and have upgraded the status, sorry!

**Comment by Chman** [comment]
You can't really delete something that's never been there from the beginning, that's my point (because you said sections of v1 docs where deleted in your original post).

The restrictions/known issues list has never been put in the v2 quickstart guide because it's still in beta and heavy development. You're not supposed to use it for production yet. And it's moving fast. Keeping up a precise list of issues & restrictions at this point is pointless. And to be honest I'm not even sure these lists in the v1 "doc" are up to date.

As for:

> To be clear: Unity's official pages on VR link directly to this wiki.

That shouldn't be the case though. I'll talk to the doc team.

**Comment by adamgit** [comment]
Ok, that makes more sense.

However ... In the vacuum of this situation, people are using this anyway 
(partly because UT is encouraging? us to!), and sharing statements about 
what does and does not work in VR, based on your current docs (whatever you 
want to call them, they are the details facto documentation that exists 
right now, as far as I can tell).

Given its a wiki, I'd expect it to be easy to keep restrictions / main 
issues up to date. For any other issues, you could hotlink to the Issues 
tab on GH, but if there are "don't report this bug, we know already!" (Eg 
"x doesn't work in VR at all, we know"), then it's worth wiki ing them so 
that ppl don't spam you with dupes.

But if the restrictions etc can't be kept up to date, then where should ppl 
check for this info? Or do you want to to keep it all private? Or do you 
prefer ppl create Issues for problems you already know about?

All those options could work, but you need to pick one, and the  we can go 
along with it :).

**Comment by Chman** [comment]
Well, that's why there's a search feature in the issue section of Github :) And people do use it, some issues are bumped by various people once in a while and we rarely get dups (except from people who don't really look for them, and they don't look at the wiki either anyway).

So at some point it's all about "is it worth it right now?". All the post-processing stuff is handled by a very tiny set of people and because v2 is still WIP and not released, we'd rather pump out features and focus on making everything work before we spare some time on documentation (which will come regardless, it's just not top priority right now). And for known issues & restrictions you already have the issue page, although I agree it's not the best it's better than nothing, listing them one by one on the wiki would just be redundant when you can already search for them.

tl;dr We do agree that there's a clear lack of documentation on Post-processing, and we do intend to solve this once v2 is ready for production but right now we're short on time so it's kinda low priority (unfortunately).
